instrumentation/otel-webserver-module/Dockerfile

```diff
@@ -219,19 +214,20 @@ RUN mkdir build-dependencies \
     && mv -f httpd-2.4.23 build-dependencies
 
 # Build Webserver Module
-RUN git clone https://github.com/open-telemetry/opentelemetry-cpp-contrib.git \
-    && cp -r /opentelemetry-cpp-contrib/instrumentation/otel-webserver-module . \
-    && cp -r /dependencies /otel-webserver-module/ \
+COPY . /otel-webserver-module
+
+RUN cp -r /dependencies /otel-webserver-module/ \
     && cp -r /build-dependencies /otel-webserver-module/ \
     && cd otel-webserver-module \
-    && ./gradlew assembleApacheModule
+    && ./gradlew assembleApacheModule -DtargetSystem=centos
```

Comment:
The default value of targetSystem is "centos6".  Since this is default, we no longer need this option here.

Reply:
Resolved